tensorflow_addons/rnn/layer_norm_lstm_cell.py

```diff
@@ -154,9 +154,16 @@ def __init__(
 
     def build(self, input_shape):
         super().build(input_shape)
-        self.kernel_norm.build([input_shape[0], self.units * 4])
-        self.recurrent_norm.build([input_shape[0], self.units * 4])
-        self.state_norm.build([input_shape[0], self.units])
+
+        def maybe_build_sublayer(sublayer, build_shape):
+            if not sublayer.built:
+                with tf.keras.backend.name_scope(sublayer.name):
```

Comment:
I've not checked the internals but what is the difference with `maybe_init_scope` at https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/engine/base_layer.py#L2707

Reply:
Usually the init scope is not needed by the cell since the RNN layer that wrap the cell should already have the correct context before the cell.build() is invoked (eg by __call__() of the model, or the layer). Manually invoke the build() method is not ideal and often error-prone if the context is not setup properly.